DevExtreme.AspNet.TagHelpers.Generator/PropTypeInfo.cs

```diff
@@ -19,15 +20,17 @@ const string
             SPECIAL_RAW_STRING = "sp_raw_string",
             SPECIAL_DOM_TEMPLATE = "sp_dom_template";
 
-        static readonly IDictionary<string, string> _overrideTable = InitOverrideTable();
+        static readonly IDictionary<string, string> _predefinedTypes = InitPredefinedTypes();
```

Comment:
I guess this rename wasn't necessary in the context of this changeset? Makes harder to read diffs.